### PR TITLE
Assignment 2 Submission

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: AjayKumarMK/assignment2/django_blog/blog/templates/blog.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+    <div class="new-body">
+        <div class="card" >
+          <div>
+            <a href="create/" class="btn btn-outline-primary">Create New Blog</a>
+            <a href="/signout" class="btn btn-outline-danger" style="float: right;">Logout</a>
+          </div>
+        </div>
+        
+        {% for blogs in blogs %}
```
Comment (friskycodeur) [inline AjayKumarMK/assignment2/django_blog/blog/templates/blog.html]: Naming convention is not right , can you change it to for blog in blogs?
Also change it down below where blogs.property has been used

File: AjayKumarMK/assignment2/django_blog/blog/models.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+class blog(models.Model):
+    title = models.CharField(max_length=200)
+    content = models.TextField()
+    date_posted = models.DateTimeField(default=timezone.now)
+
+    user_id = models.ForeignKey(User,on_delete=models.CASCADE,null=True,blank=True)
```
Comment (friskycodeur) [inline AjayKumarMK/assignment2/django_blog/blog/models.py]: This field should not be nullable, can you fix this?

File: AjayKumarMK/assignment2/django_blog/blog/templates/posts.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    </div>
+    <div class="new-body">
+        <div class="card">
+            <div>
+                <a href="/blogs" class="btn btn-outline-primary">Back</a>
+                <a href="/signout" class="btn btn-outline-danger" style="float: right;">Logout</a>
+            </div>
+        </div>
+        <div class="card">
+            <h2 class="card-title">{{ blogs.title }}</h2>
```
Comment (friskycodeur) [inline AjayKumarMK/assignment2/django_blog/blog/templates/posts.html]: Shouldn't you use a loop, like for blog in blogs?

File: AjayKumarMK/assignment2/django_blog/user/templates/signup.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```
Comment (friskycodeur) [inline AjayKumarMK/assignment2/django_blog/user/templates/signup.html]: Use Template inheritance, rather than writing the whole code again.

**Review by friskycodeur — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@friskycodeur  Completed the changes.

**Comment by friskycodeur** [comment]
Great job @krrishhrx

**Pull request merged**